packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -22,6 +23,30 @@ import {
 } from './InternalHelpers';
 import { IHttpNegotiationResult, NegotiatePartialOptions, NegotiationOptions } from './types';
 
+export const warnMsg = (contentTypes: string[]) => `Unable to find content for ${contentTypes}`;
+
+const whenNoContent = (
+  logger: Logger,
+  response: IHttpOperationResponse,
+  headers: IHttpHeaderParam[],
+  mediaTypes: string[],
```

Comment:
I would really specify the return type explicitly — it helps the readability but more importantly the type checker will help us in case we're going to refactor this function, effectively disabling the inference.